### DISCUSSION TITLE
Choose location for `.coverage` file

### DISCUSSION THREAD

**Discussion by dgard1981** [pytest-dev/pytest#11263] [Q&A] [answered]
When using `pytest-cov` with a test run, the file ***.coverage*** is created in the root directory of the run. Is it possible to specify the location of this file using either CLI options or settings in `.coveragerc`?

I suspect that CLI options don't offer this facility as the are fairly minimal and [well documented](https://pytest-cov.readthedocs.io/en/latest/config.html).

However, it's possible I'm missing something in the [Configuration Reference](https://coverage.readthedocs.io/en/latest/config.html), but I can't see anything that obviously stands out.

Thanks.

**Comment by The-Compiler** [comment, marked as answer]
> However, it's possible I'm missing something in the [Configuration Reference](https://coverage.readthedocs.io/en/latest/config.html), but I can't see anything that obviously stands out.

From a quick search for `.coverage` on that page:

> ### [[run] data_file](https://coverage.readthedocs.io/en/latest/config.html#run-data-file)
> (string, default “.coverage”) The name of the data file to use for storing or reporting coverage. This value can include a path to another directory.

**Reply by dgard1981** [reply]
Thank you, I missed that on the page.
